subprojects/problems/src/main/java/org/gradle/api/problems/interfaces/Problem.java

```diff
@@ -37,9 +37,6 @@ public interface Problem {
 
     Severity getSeverity();
 
-    @Nullable
-    ProblemLocation getWhere();
```

Comment:
❌ The idea is to remove the location from the TAPI data structures but keep it in the internal API.

We do want to have this information stored in the Problem instances (for rendering) and add it to the internal build operation progress events

Reply:
Redone, PTAL